### PR TITLE
fix: log error & remove deprecated linter

### PR DESCRIPTION
**Pull request by carr0t2**
Such as input the wrong command line parameters
The original error will be panic since log is not init.
image
After fix.
image

### CODE REVIEW

**Comment by carr0t2** [comment]
The linter 'varcheck' 'deadcode' 'structcheck' is deprecated (since v1.49.0), use 'unused' instead.

**Comment by moonD4rk** [comment]
> The linter 'varcheck' 'deadcode' 'structcheck' is deprecated (since v1.49.0), use 'unused' instead.

THX, I will remove this lint, you could revert lint commit~

**Review by moonD4rk — CHANGES_REQUESTED** [review]
revert golangci config file, and change log to `Println` and `os.Exit`, Thx

**Review by moonD4rk — APPROVED** [review]

File: cmd/hack-browser-data/main.go [hunk 1]
```diff
@@ -72,6 +72,7 @@ func Execute() {
 	}
 	err := app.Run(os.Args)
 	if err != nil {
+		log.Init("debug")
```
[the remark this answers is not in the document]
Comment (moonD4rk) [reply cmd/hack-browser-data/main.go:72]: This is better, so i changed your code and merge to master


```go
import (
	glog "log"
)
func main() {
	glog.fatal(err)
}
```

**Pull request merged**